### ISSUE TITLE
Enable Bookmark selection from Search Screen

### ISSUE THREAD

**Issue by Starwer** [mapsme/omim#9313]
It takes quite some learning to just plan a route from one place in my bookmarks to another place in my bookmarks, although the usecase seems trivial.

In the Route Planning screen, when selecting (1) "Your Location"  or (2) "To", you get to the Search-screen proposing to search, to access history or to search by category. This makes sense, but the this is counter intuitive in the presented usecase.

What would help a great deal would be to add to the Search screen a tab "Bookmarks" (in addition to the "History" and  "Categories") to enable bookmark selection there. This way, you make the Bookmarks available everywhere the search is available.

**Comment by Starwer** [comment]
@burivuh: I'm not trying to add a bookmark here, but trying to leverage the set of existing bookmarks. 
 Let me rephrase my problem description with a simple usecase...
How can I know, for example, the time it will take me to drive from one spot in my bookmark to another spot in my bookmark?
This is a basic when planning any road trip...

**Comment by burivuh** [comment]
I'm talking about adding to the route, not creating a new one.
We have two controls on the side of the map to the left to select objects. One - through search, another - through bookmark selection.

It's not as simple as ability to select bookmark using our tips panel at the bottom, I agree, but overall you do have a method to select bookmarks and add them to the route.

**Comment by Starwer** [comment]
Thanks @burivuh ! I'll surely use this trick... 

It's indeed not as simple as what I proposed, and also not as fast as Maps.me seems to recompute the all road (A-B-C) instead of just computing from B to C. 
Until there is a more intuitive way to do this I'll just recommend to my parents to use google map for this kind of things....

**Comment by WiliTest** [comment]
I agree, in google map, i just open the "find" tool, and search "Home"  (the name of a bookmark likn to an adress), and it goes there. I mostly use google map simply because of that.
